sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -1979,7 +1979,8 @@ class Dataset[T] private[sql](
   * {{{
   *   // Monitor the metrics using a listener.
   *   spark.streams.addListener(new StreamingQueryListener() {
-  *     override def onQueryProgress(event: QueryProgressEvent): Unit = {
```

Comment:
@HyukjinKwon Thank you for the ping. Does the origin example cannot work ?

Reply:
oh! it was my mistake. It didn't follow the order. let me correct a bit